canonical-web-and-design/ubuntu.com
# Initial cut of 19.10 release variables and download pages

## Done

- Updated the release variables
- Fixed the server download to look like the desktop download
- Removed the pictogram from the flavours page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/download/server, http://0.0.0.0:8001/download/desktop, http://0.0.0.0:8001/download/alternative-downloads and http://0.0.0.0:8001/download/flavours
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that it offers 19.10 over 19.04 and OpenStack Train


## Issue / Card

Fixes #5849
